tests/e2e/config/default.json

```diff
@@ -1,6 +1,5 @@
 {
   "url": "http://localhost:8084/",
-  "appName": "woocommerce_e2e",
```

Comment:
@veljkho This wasn't the correct file. Can you revert it? I commented on `tests/e2e/env/config/default.json` vs `tests/e2e/config/default.json`. The `env` change was a result of the issue fixed in #29060.